### ISSUE TITLE
Simulation of data with items with different number of categories

### ISSUE THREAD

**Issue by jeppebundsgaard** [alexanderrobitzsch/TAM#16]
I wanted to simulate a model I produced, but ran into something I think is connected to the number of categories. 
Here is a minimal working example: The first one works, the next one gives a lot of NA's in the simulated data...:

```
data<-data.frame(i1=c(0,1,0,1,0,1),i2=c(0,1,1,0,1,1))
mod<-tam.mml(data)
sim<-IRT.simulate(mod)
sim

data<-data.frame(i1=c(0,1,0,1,0,1),i2=c(0,1,2,0,1,2))
mod<-tam.mml(data)
sim<-IRT.simulate(mod)
sim
```

**Comment by thkiefer** [comment]
I can replicate that behavior and proposed a fix.

**Comment by alexanderrobitzsch** [comment]
Thank you. I released a new version ``TAM 3.6-39`` that includes the bug fix.

**Comment by jeppebundsgaard** [comment]
Thank you both for the very swift solution. It also works on my real life data set. I can continue my analyses. Thank you very much!
